client/src/components/BadReview/BadReviewControllers.tsx

```diff
@@ -38,6 +38,12 @@ export function BadReviewControllers({ courseTasks, courseId }: IBadReviewContro
     }
   }, [taskId, checkType, checkService, courseId]);
 
+  const downloadSolutionsUrls = async () => {
+    if (courseId && taskId) {
```

Comment:
is this check really needed? I think we should have it always here. if so consider making button a link with url

Reply:
We need check, because task may be not selected. But I like idea to have button with link instead of this check